addons/im_livechat/tests/test_get_discuss_channel.py

```diff
@@ -208,6 +209,7 @@ def test_channel_get_livechat_visitor_info(self):
                     "is_public": False,
                     "name": "Michel",
                     "notification_preference": "email",
+                    "signature": str(operator.signature),
```

Comment:
re-check those, should probably not have operator signature

Reply:
Yes sorry, what's happening in this test case the operator is testing chatting themselves so they receive their own signature as expected.

Reply:
What would have been strange is if the visitor received the operator signature, but that's not the case here.